libsrc/telestrat/kbhit.s

```diff
@@ -6,12 +6,21 @@
 
         .export         _kbhit
 
+        .export         store_char
+
         .include        "telestrat.inc"
 
 _kbhit:
         BRK_TELEMON XRD0
+
         ldx     #$00
-        txa
-        rol
-        eor     #$01
+        bcs     @no_char_action
+        sta     store_char
+        lda     #$01
+        rts
+@no_char_action:
+        tax
         rts
+.data
+store_char:
```

Comment:
If you move that variable to "cgetc.s", then `kbhit()` won't be linked into programs if it actually isn't called.

("samples/ascii.c" calls `cgetc()` and doesn't call `kbhit()`.)